src/Components/AddBook.js

```diff
@@ -30,27 +29,32 @@ const AddBook = () => {
   };
 
   return (
-    <form className="form">
-      <input
-        type="text"
-        placeholder="Title"
-        name="title"
-        required
-        value={newBook.title}
-        onChange={handleInput}
-      />
-      <input
-        type="text"
-        placeholder="Author"
-        name="author"
-        required
-        value={newBook.author}
-        onChange={handleInput}
-      />
-      <button type="submit" className="add-btn" onClick={handleClick}>
-        Add
-      </button>
-    </form>
+    <div className="form-div">
+      <h3 className="add-new-book">Add New Book</h3>
+      <form className="add-form">
+        <input
+          className="input"
+          type="text"
+          placeholder="Title"
+          name="title"
+          required
+          value={newBook.title}
+          onChange={handleInput}
+        />
+        <select className="input" style={{ width: '165px' }}>
+          <option value="" disabled>
```

Comment:
- [ ] No inline styling (Note: React inline styling is allowed but a variable should be created and passed to React style. Let's fix that
Here is an example

```js
const myComponentStyle = {
  color: 'blue',
  lineHeight: 10,
  padding: '1.5em',
}

function MyComponent(){
    return <div style={myComponentStyle}> Inline Styled Component</div>
}
```

Reply:
Hi @EricMbouwe thank you for your feedback. I have removed the inline styling